### PR TITLE
Cards: Header padding-top fix

### PR DESCRIPTION
**Pull request by louismaximepiton**
Not sure that the actual implementation is the best but if we think for the future and the small and large cards will change this value as well.

### CODE REVIEW

**Review by MewenLeHo — APPROVED** [review]
🆗

File: scss/_card.scss [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 .card {
   // scss-docs-start card-css-vars
+  --#{$prefix}card-spacer-top: #{$card-spacer-top}; // Boosted mod
```
Comment (julien-deramond) [inline scss/_card.scss:7]: IMO it's disturbing to have a `--bs-card-spacer-top` that acts on the top padding and then something named `--bs-card-spacer-y` only used for the bottom padding.
As a user, I don't know how to use those variables; will `--bs-card-spacer-top` override the value of the `--bs-card-spacer-y`, which one has the priority, etc.

Comment (louismaximepiton) [reply scss/_card.scss:7]: Another solution could be to set the padding top as a ratio of the `card-spacer-y`

File: site/content/docs/5.2/migration.md [hunk 2]
```diff
@@ -132,9 +133,18 @@ If you need more details about the changes, please refer to the [v5.2.3 release]
     </ul>
   </details>
 
+- <details class="mb-2">
+    <summary><span class="badge bg-danger">Breaking</span> Dropped Sass variables:</summary>
```
Comment (julien-deramond) [inline site/content/docs/5.2/migration.md:137]: Note for myself: replace "Dropped" by "Deprecated" in this migration guide

**Review by julien-deramond — APPROVED** [review]
LGTM!

**Pull request merged**